charts/values.yaml

```diff
@@ -0,0 +1,137 @@
+# Default values for kubeovn.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+networking:
+  # net_stack could be dual_stack, ipv4, ipv6
+  net_stack: ipv4
+  enable_ssl: false
+  network_type: geneve
+  IFACE: ""
+  DPDK_TUNNEL_IFACE: "br-phy"
+  EXCLUDE_IPS: "" 
+  POD_NIC_TYPE: "veth-pair"
+  vlan:
+    VLAN_INTERFACE_NAME: ""
+    VLAN_NAME: "ovn-vlan"
+    VLAN_ID: "100"
+
+func:
+  ENABLE_LB: true
+  ENABLE_NP: true
+  ENABLE_EIP_SNAT: true
+  ENABLE_EXTERNAL_VPC: true
+  HW_OFFLOAD: false
+
+ipv4:
+  POD_CIDR: "10.16.0.0/16"
+  POD_GATEWAY: "10.16.0.1"
+  SVC_CIDR: "10.96.0.0/12"   
+  JOIN_CIDR: "100.64.0.0/16"    
+  PINGER_EXTERNAL_ADDRESS: "114.114.114.114" 
+  PINGER_EXTERNAL_DOMAIN: "alauda.cn"  
+
+ipv6:
+  POD_CIDR: "fd00:10:16::/64"    
+  POD_GATEWAY: "fd00:10:16::1"
+  SVC_CIDR: "fd00:10:96::/112"   
+  JOIN_CIDR: "fd00:100:64::/64"     
+  PINGER_EXTERNAL_ADDRESS: "2400:3200::1"
+  PINGER_EXTERNAL_DOMAIN: "google.com"
+
+dual_stack:
+  POD_CIDR: "10.16.0.0/16,fd00:10:16::/64"   
+  POD_GATEWAY: "10.16.0.1,fd00:10:16::1"
+  SVC_CIDR: "10.96.0.0/12,fd00:10:96::/112"     
+  JOIN_CIDR: "100.64.0.0/16,fd00:100:64::/64"    
+  PINGER_EXTERNAL_ADDRESS: "114.114.114.114,2400:3200::1"
+  PINGER_EXTERNAL_DOMAIN: "google.com"
+
+image:
+  repository: "kubeovn"
+  pullPolicy: IfNotPresent
+  version: "v1.9.2" 
+
+perform:
+  MODULES: "kube_ovn_fastpath.ko"
+  RPMS: "openvswitch-kmod"
+
+debug:
+  ENABLE_MIRROR: false
+
+cni_conf:
+  MASTER_NODES: ""
+  CHECK_GATEWAY: true
+  LOGICAL_GATEWAY: false
+  CNI_CONFIG_PRIORITY: "01"
+  CNI_CONF_DIR: "/etc/cni/net.d"
+  CNI_BIN_DIR: "/opt/cni/bin"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+ingress:
```

Comment:
if these values are not used, do not represent them in values.yaml

Reply:
done